docs/current_docs/cookbook/cookbook.mdx

```diff
@@ -193,6 +197,51 @@ The following Dagger Function accepts a `Directory` argument, which could refere
       entries
     ```
 
+### Copy a subset of a directory or remote repository to a container using pre-defined filters
+
+The following Dagger Function accepts a `Directory` argument, which could reference either a directory from the local filesystem or from a remote Git repository. It copies the specified directory to the `/src` path in a container, using pre-defined filter patterns to exclude specified sub-directories and files, and returns the modified container.
```

Comment:
Did you write somewhere what's the difference between post & pre filtering? And which one is better depending on the case?

Reply:
The cookbook is deliberately light on explanation, so I have added a link to the more detailed docs page discussing both pre- and post-call filters.